docs/source/development/defining_new_models.md

```diff
@@ -217,23 +255,3 @@ def schema() -> dict:
 
     return config_schema
 ```
```

Comment:
Is this code the same for all models, except for the name of the schema file?

If so, you could remove some of the boilerplate and just do something like this in all the `__init__.py` files:

```py
from virtual_rainforest.core.config import default_schema_setup

schema = default_schema_setup()
```

You'll probably need to examine the stack for this.

Reply:
It is the same for all models. That's a good thought.